LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -53,22 +53,24 @@
                 <div id="groupList" class="leaf-displayFlexRow"></div>
             </div>
         </div>
-
     </main>
-
-
 </div>
+<!--Loading Modal-->
+<!--{include file="../../../libs/smarty/loading_spinner.tpl"}-->
```

Comment:
Might want to see if you can include a variable here in the include tag that is like spinner-name="User Groups" and then in the below template you could put {{spinner-name}} to have it output what you passed in.  Makes it more extendable.  Just a thought.

Reply:
Ah yeah, you are right I overlooked this.